views/sign-out.html

```diff
@@ -0,0 +1,57 @@
+{% extends "layout.html" %}
```

Comment:
Just wondering, does the sign-out link appear on the 'confirm sign out' page too? Should it?

Reply:
It does not appear on the new sign out page.